Packs/qualys/Integrations/Qualysv2/Qualysv2.py

```diff
@@ -34,6 +34,7 @@
 ASSET_SIZE_LIMIT = 10 ** 6   # 1MB
 TEST_FROM_DATE = 'one day'
 FETCH_ASSETS_COMMAND_TIME_OUT = 180
+QIDS_BATCH_SIZE = 2000
```

Comment:
@kamalq97 , i know this was mentioned in the ticket but it was somewhat arbitrary to me (i suggested it).  I tested that API with postman with 2k QIDs and it works but can still take 1.5 minutes.  Just more of an FYI than anything.

Reply:
Lowered the value to 500 in commit b64fb6bcc4be2f396c213f3a4a3261f2abb197b2 following our private discussion.

This seems to yield a good compromise between API response time and number of API calls.